platforms/native/engine/jsb-loader.js

```diff
@@ -55,7 +55,7 @@ function downloadScript (url, options, onComplete) {
     if (loadedScripts[url]) return onComplete && onComplete();
 
     download(url, (src, options, onComplete) => {
-        if (window.oh) {
+        if (window.oh && window.scriptEngineType === 'napi') {
```

Comment:
do we need to add the scriptEngineType defined
https://github.com/cocos/cocos-engine/blob/9f9ee38427b42db7086e689fe568252e06d623ad/%40types/jsb.d.ts#L372-L376

Reply:
It's a property in `global` rather than in `jsb` namespace.

Reply:
I think we could not add a new property in global namespace, right?

Reply:
right